pkg/oc/cli/describe/chaindescriber_test.go

```diff
@@ -197,53 +199,55 @@ func TestChainDescriber(t *testing.T) {
 		},
 	}
 
-	for _, test := range tests {
-		objs := []runtime.Object{}
-		if len(test.path) > 0 {
-			var err error
-			objs, err = readObjectsFromPath(test.path, test.defaultNamespace, legacyscheme.Codecs.UniversalDecoder(), legacyscheme.Scheme)
-			if err != nil {
-				t.Fatal(err)
+	for i, test := range tests {
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
+			objs := []runtime.Object{}
+			if len(test.path) > 0 {
+				var err error
+				objs, err = readObjectsFromPath(test.path, test.defaultNamespace, unstructured.UnstructuredJSONScheme, legacyscheme.Scheme)
```

Comment:
I'd be surprised if this worked.

Reply:
Well, with the UniversalDecoder I'm getting unstructured types, in which case the rest of functionality fails, that's why I switched to UnstructuredJSON

Reply:
Fixed properly.